application/config/config.go

```diff
@@ -561,7 +568,13 @@ func getCodeApiUrlFromCustomEndpoint(endpoint string) (string, error) {
 	}
 
 	m := regexp.MustCompile(`^(ap[pi]\.)?`)
-	endpointUrl.Host = m.ReplaceAllString(endpointUrl.Host, "deeproxy.")
+
+	subdomain := "deeproxy."
+	if isFedramp {
+		subdomain = "api."
```

Comment:
That might not work if a user provides a custom endpoint as `https://api.snykgov.io` in their settings (updated via [here](https://github.com/snyk/snyk-ls/blob/adc52f1731a191eef1b398ff3cffe6c1d1973224/application/config/config.go#L376)).